src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/jpa/entity/TierCalculationEntityTest.kt

```diff
@@ -14,17 +14,22 @@ internal class TierCalculationEntityTest {
   @Test
   fun `Should Construct TierCalculationEntity`() {
     val crn = "Any CRN"
+    val id = 1L
     val created = LocalDateTime.now()
     val data = data
     val calculationId = UUID.randomUUID()
 
-    val tierCalculationResultEntity = TierCalculationEntity(crn = crn, created = created, data = data, uuid = calculationId)
+    val tierCalculationResultEntity = TierCalculationEntity(id, calculationId, crn, created, data)
+    val tierCalculationResultEntity2 = TierCalculationEntity(id, calculationId, crn, created, data)
 
     assertThat(tierCalculationResultEntity.crn).isEqualTo(crn)
     assertThat(tierCalculationResultEntity.created).isEqualTo(created)
     assertThat(tierCalculationResultEntity.data).isEqualTo(data)
-
     assertThat(tierCalculationResultEntity.uuid).isEqualTo(calculationId)
+    assertThat(tierCalculationResultEntity.id).isEqualTo(id)
+    assertThat(tierCalculationResultEntity).isEqualTo(tierCalculationResultEntity2)
+    assertThat(tierCalculationResultEntity.toString()).isEqualTo(tierCalculationResultEntity2.toString())
```

Comment:
this test is now testing two things, if you wanted to test this write another test